### PR TITLE
test(api): add getHealth unit tests

### PR DESCRIPTION
**Pull request by abdollahShahid**
Part of fossology#1590

## Summary
This PR adds PHPUnit unit tests for the REST API `InfoController::getHealth()` endpoint.
This is **Part 1** of an incremental effort to improve REST API test coverage while keeping review scope small.

---

## Changes
- Added unit tests for `InfoController::getHealth()`
- Covered healthy responses and error/exception scenarios
- Adjusted `phpunit.xml` to run these tests in the current setup

---

## Tested locally
- [x] `InfoControllerTest` passes locally
- [x] No existing tests are broken (only test/config changes)
- [x] No production code paths were modified

---

## How to test 
From the repository root:

```bash
cd src
php .\vendor\bin\phpunit --configuration .\phpunit.xml --filter InfoControllerTest
```
Expected result:

OK (6 tests, 18 assertions)

## Notes for maintainers

This PR is intentionally limited to one REST endpoint to keep review scope small.
Additional REST API endpoints and edge cases will be covered in follow-up PR(s).
The issue fossology#1590 will be fully resolved once all parts are merged.

### CODE REVIEW

File: src/phpunit.xml [hunk 1]
```diff
@@ -9,20 +9,10 @@
...
-  cacheDirectory=".phpunit.cache">
-  <coverage>
-    <include>
-      <directory suffix=".php">.</directory>
-    </include>
-    <report>
-      <clover outputFile="../clover.xml"/>
-    </report>
-  </coverage>
+  xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.0/phpunit.xsd">
```
[the remark this answers is not in the document]
Comment (abdollahShahid) [reply src/phpunit.xml:12]: The changes to phpunit.xml were unintentional and have been reverted to match the upstream configuration.
This PR is now limited strictly to adding unit tests for getHealth() to keep the review scope small.
